src/Files.App/App.xaml

```diff
@@ -2,7 +2,8 @@
 <Application
 	x:Class="Files.App.App"
 	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	HighContrastAdjustment="None">
```

Comment:
What does this property do?

Reply:
> What does this property do?

https://learn.microsoft.com/en-gb/uwp/api/windows.ui.xaml.uielement.highcontrastadjustment?view=winrt-22621

Reply:
written in the remarks section.

Reply:
@mdtauk adding it in the app.xaml doesn't work. so added each page, i confirmed they are at work.

Reply:
![image](https://github.com/files-community/Files/assets/62196528/c8e792cf-1d4c-4d88-af6f-ce7050806833)

Reply:
Let's hold off on this change, I think we need to do some testing to see how it affects the rest of the app.

Reply:
we can test in this PR. This PR's purpose is to improve visual elements and so I think it is not out of the scope of the PR.

Reply:
And I tested in the contents area and the settings area. The controls from MUXC like TabView has already applied this property in the custom style ([custom TabView style](https://github.com/files-community/Files/blob/main/src/Files.App/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml#L218)).

Reply:
> @mdtauk adding it in the app.xaml doesn't work. so added each page, i confirmed they are at work.

This seems to be a bug, do you know if it's reported on the WinUI repo yet?